index.js

```diff
@@ -133,6 +133,7 @@ const ruleFunction = (expectation, options, context) => {
         message,
         node: root,
         index: getIndexFromLoc(source, err.loc.start),
+        endIndex: getIndexFromLoc(source, err.loc.start + result.css.length),
       });
```

Comment:
we do not have `erro.loc.end`, but we have `length` of content, this can be `endIndex`